README.md

```diff
@@ -45,7 +45,7 @@ with cross-platform Python packaging.
 Initialized a new project in <>
 $ cp ../cloudformation-cli-python-lib-0.0.1.tar.gz .
 $ cfn submit --dry-run
-$ cat test.json
+$ cat <<EOT > test.json
```

Comment:
Is this necessary?

Reply:
Yeah. if the file `test.json` doesn't exist `cat test.json` fails and the the subsequent lines are interpreted as commands instead of text for the file. Using the [heredoc](https://en.wikipedia.org/wiki/Here_document#Unix_shells) format fixes that.

````bash
CrossAccountRole-RootRole-1AYURKOCV4R8Y:~/environment $ cat test.json
cat: test.json: No such file or directory
CrossAccountRole-RootRole-1AYURKOCV4R8Y:~/environment $ {
>   "credentials": {
>     "accessKeyId": "",
>     "secretAccessKey": "",
>     "sessionToken": ""
>   },
>   "action": "CREATE",
>   "request": {
>     "clientRequestToken": "ecba020e-b2e6-4742-a7d0-8a06ae7c4b2b",
>     "desiredResourceState": {
>       "Title": "foo",
>       "Description": "bar"
>     },
>     "previousResourceState": null,
>     "logicalResourceIdentifier": null
>   },
>   "callbackContext": null
> }
bash: credentials:: command not found
bash: accessKeyId:: command not found
bash: secretAccessKey:: command not found
bash: sessionToken:: command not found
bash: },: command not found
bash: action:: command not found
bash: request:: command not found
bash: clientRequestToken:: command not found
bash: desiredResourceState:: command not found
bash: Title:: command not found
bash: Description:: command not found
bash: },: command not found
bash: previousResourceState:: command not found
bash: logicalResourceIdentifier:: command not found
bash: },: command not found
bash: callbackContext:: command not found
CrossAccountRole-RootRole-1AYURKOCV4R8Y:~/environment $ ls
README.md
````